dll/win32/msutb/lang/pt-PT.rc

```diff
@@ -0,0 +1,79 @@
+/*
+ * PROJECT:     ReactOS msutb.dll
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2023-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
+ */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
+
+STRINGTABLE
+BEGIN
+    IDS_OK               "&OK"
+    IDS_CANCEL           "&Cancelar"
+    IDS_ABORT            "&Abortar"
+    IDS_RETRY            "&Retroceder"
+    IDS_IGNORE           "&Ignorar"
+    IDS_YES              "&Sim"
+    IDS_NO               "&Não"
+
+    IDS_SETTINGS         "&Definições..."
+    IDS_RESTORELANGBAR   "Restaurar a barra de idiomas"
+    IDS_LANGUAGE         "Linguagem"
+    IDS_LANGUAGEBUTTON   "Botão Linguagem"
+    IDS_TRANSPARENCY     "&Transparência"
+    IDS_TEXTLABELS       "T&exto da etiqueta"
+    IDS_MINIMIZE         "&Minimizar"
+    IDS_CLOSELANGBAR     "&Fechar a barra de idiomas"
+    IDS_EXTRAICONS       "&Ícones adicionais na barra de tarefas"
```

Comment:
@zecarlos1957 Is accentued I a good keyboard accelerator? wouldn't this require the user to press more than two keys (the accent, the capital I, and Alt)?